frontend/src/__tests__/cypress/cypress/tests/e2e/pipelines/pipelines.cy.ts

```diff
@@ -0,0 +1,194 @@
+import {
+  createOpenShiftProject,
+  deleteOpenShiftProject,
+  applyOpenShiftYaml,
+} from '~/__tests__/cypress/cypress/utils/ocCommands';
+import { replacePlaceholdersInYaml } from '~/__tests__/cypress/cypress/utils/yaml_files';
+import { ADMIN_USER } from '~/__tests__/cypress/cypress/utils/e2eUsers';
+import { AWS_PIPELINES_BUCKET } from '~/__tests__/cypress/cypress/utils/s3Buckets';
+
+import { projectListPage, projectDetails } from '~/__tests__/cypress/cypress/pages/projects';
+import { pipelineImportModal } from '~/__tests__/cypress/cypress/pages/pipelines/pipelineImportModal';
+import { pipelinesGlobal } from '~/__tests__/cypress/cypress/pages/pipelines/pipelinesGlobal';
+import { pipelinesTable } from '~/__tests__/cypress/cypress/pages/pipelines/pipelinesTable';
+import { createRunPage } from '~/__tests__/cypress/cypress/pages/pipelines/createRunPage';
+import { pipelineRunDetails } from '~/__tests__/cypress/cypress/pages/pipelines/topology';
+
+const projectName = 'test-pipelines-prj';
+const dspaSecretName = 'dashboard-dspa-secret';
+const allAvailableProjectsText = 'All available projects';
+const testPipelineName = 'test-pipelines-pipeline';
+const testRunName = 'test-pipelines-run';
+
+describe('An admin user can import and run a pipeline', { testIsolation: false }, () => {
+  before(() => {
+    // Provision a Project
+    createOpenShiftProject(projectName).then((result) => {
+      expect(result.code).to.eq(
+        0,
+        `ERROR provisioning ${projectName} Project
+                                    stdout: ${result.stdout}
+                                    stderr: ${result.stderr}`,
+      );
```

Comment:
All this error handling can be part of the underlying function instead of repeating each time.

Reply:
Fixed in [4d9a76b](https://github.com/opendatahub-io/odh-dashboard/pull/3028/commits/4d9a76b0f90b4a6b104a784dd800b2cd2f40bf91)